README.md

```diff
@@ -32,6 +32,14 @@ If you don't already have it, install [.NET 8+ SDK](https://dotnet.microsoft.com
 
 * `dotnet new console -o MyApp`
 * `cd MyApp`
+
+Create a `nuget.config` file and add the `dotnet-experimental` package source for the `NativeAOT-LLVM` dependency:
+
+* `dotnet new nugetconfig`
+* Add the key to `nuget.config` after `<clear />`: `<add key="dotnet-experimental" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-experimental/nuget/v3/index.json" />`
```

Comment:
if we add the clear, I think we need to re-include `    <add key="nuget" value="https://api.nuget.org/v3/index.json" />`

Reply:
@jsturtevant Ah, thanks - updated!